### DISCUSSION TITLE
D792FBEE-4FB2-402E-8CE0-22021AFE12CE

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#190] [General]
# D792FBEE-4FB2-402E-8CE0-22021AFE12CE

Kepler-52, also known as KOI-775, is an M-type main sequence star located 1048.74 light-years away from Earth. This star has three planets (one rocky planet and two mini Neptune planets) with radii larger than Earth, and the outermost planet has a orbital period of only 36.44 days

https://celestia.mobi/addon?item=D792FBEE-4FB2-402E-8CE0-22021AFE12CE&lang=zh_CN

**Comment by chant-19OX** [comment]
非常不错的插件！
A nice addon
